include/OpenSpaceToolkit/Astrodynamics/Trajectory/CoordinatesSubset.hpp

```diff
@@ -0,0 +1,80 @@
+/// Apache License 2.0
+
+#ifndef __OpenSpaceToolkit_Astrodynamics_Trajectory_CoordinatesSubset__
+#define __OpenSpaceToolkit_Astrodynamics_Trajectory_CoordinatesSubset__
+
+#include <OpenSpaceToolkit/Core/Types/Integer.hpp>
+#include <OpenSpaceToolkit/Core/Types/String.hpp>
+
+namespace ostk
+{
+namespace astro
+{
+namespace trajectory
+{
+
+using ostk::core::types::Integer;
+using ostk::core::types::String;
+
+/// @brief  State coordinates subset.
+
+class CoordinatesSubset
+{
+   public:
+    /// @brief              Constructor
+    ///
+    /// @code
+    ///                     CoordinatesSubset coordinatesSubset("POSITION", 3) ;
+    /// @endcode
+    ///
+    /// @param              [in] anId The coordinate subset unique identifier
+    /// @param              [in] aSize The coordinate subset size
+
+    CoordinatesSubset(const String& anId, const Integer& aSize);
```

Comment:
```suggestion
    CoordinatesSubset(const String& anId, const Size& aSize);
```

Same here. Consider using the `Size` type

Reply:
done!